client/src/reducers/user.reducers.js

```diff
@@ -0,0 +1,33 @@
+import { userConstants } from "../actions/constants";
+
+const initState = {
+    error: null,
+    message: "",
+    loading: false
+}
+
+export default ( state = initState, action ) => {
+    switch( action.type ) {
+        case userConstants.USER_REGISTER_REQUEST:
```

Comment:
not write like this, First initialized then use it
ex: `case USER_REGISTER_REQUEST`
ps:- also change above all files accordingly.

Reply:
I have declared it inside constant file